example/hello_classification_node/main.js

```diff
@@ -291,6 +247,7 @@ async function main() {
     } else {
       input_data = new Float32Array(input_blob.wmap());
```

Comment:
Why do we read as `Uint8Array` if not preprocess?

Reply:
We set `input_info.setPrecision('u8');` if not preprocess. We do preprocess in JS currently. I think we should use `PreProcessInfo` that is supported with #30 merged.

Reply:
I think I can use `PreProcessInfo` in the next PRs